src/Microsoft.ML.EntryPoints/Microsoft.ML.EntryPoints.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Add this in PropertyGroup to make it separate nuget  
  <IncludeInPackage>Microsoft.ML.EntryPoints</IncludeInPackage>